can_tools/scrapers/official/GA/ga_vaccines.py

```diff
@@ -18,39 +18,29 @@ class GeorgiaCountyVaccine(StateDashboard):
     fetch_url = "https://georgiadph.maps.arcgis.com/sharing/rest/content/items/e7378d64d3fa4bc2a67b2ea40e4748b0/data"
 
     variables = {
-        "CUMPERSONCVAX": variables.FULLY_VACCINATED_ALL,
-        "CUMPERSONVAX": variables.INITIATING_VACCINATIONS_ALL,
+        "PERSONCVAX": variables.FULLY_VACCINATED_ALL,
```

Comment:
The data in the timeseries sheets, which is what we were using previously, looks wrong. For many counties, fully-vaccinated was greater than vaccinations-initiated. 

This somewhat draws the other data in their export into question, but the "county summary" sheet (that does not have historical data) does not seem to have the same issues.

As such, this pulls from the summary sheet, which means we lose timeseries data going forward. If the GA CDC data weren't so bad, I would recommend switching over

Reply:
Ugh. :/ Thanks for digging into it and explaining.